docs/user/ppl/functions/datetime.rst

```diff
@@ -669,13 +669,13 @@ DAYOFMONTH
 Description
 >>>>>>>>>>>
 
-Usage: dayofmonth(date) extracts the day of the month for date, in the range 1 to 31. The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
```

Comment:
What happens now with zero dates? Kind of a weird old comment.

Reply:
I am taking out these lines because those are invalid dates and I see no need on them being specifically mentioned, it's like saying on time functions that seconds can't be higher than 60, it is true but also implied

Reply:
Makes sense!